<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmpleadoVentasWindow</class>
 <widget class="QMainWindow" name="EmpleadoVentasWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Panel de Ventas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="topbar">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="status_label1">
         <property name="text">
          <string>BD Remota 1: Desconectada</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label2">
         <property name="text">
          <string>BD Remota 2: Desconectada</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="logout_button">
         <property name="text">
          <string>Cerrar Sesión</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="clientes_tab">
       <attribute name="title">
        <string>Clientes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="clientes_list"/>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Documento:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="doc_cliente_edit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nombre:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nombre_cliente_edit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Teléfono:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="telefono_cliente_edit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Dirección:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="direccion_cliente_edit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Correo:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="correo_cliente_edit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="nuevo_cliente_button">
            <property name="text">
             <string>Nuevo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="guardar_cliente_button">
            <property name="text">
             <string>Guardar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="reservas_tab">
       <attribute name="title">
        <string>Reservas</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QListWidget" name="reservas_pendientes_list"/>
        </item>
        <item>
         <widget class="QPushButton" name="aprobar_reserva_button">
          <property name="text">
           <string>Aprobar Reserva</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="vehiculos_tab">
       <attribute name="title">
        <string>Vehículos</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QListWidget" name="vehiculos_disponibles_list"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="perfil_tab">
       <attribute name="title">
        <string>Mi Perfil</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Nombre:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="nombre_perfil_edit"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Teléfono:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="telefono_perfil_edit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Dirección:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="direccion_perfil_edit"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Correo:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="correo_perfil_edit"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="guardar_perfil_button">
          <property name="text">
           <string>Guardar Cambios</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="cambiar_contrasena_tab">
       <attribute name="title">
        <string>Cambiar Contraseña</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>Contraseña Actual:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="actual_pass_edit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>Nueva Contraseña:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="nueva_pass_edit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>Confirmar Nueva Contraseña:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="confirmar_pass_edit">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="cambiar_pass_button">
          <property name="text">
           <string>Cambiar Contraseña</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>